dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1123,17 +1123,22 @@ void TeachingTip::OnPopupOpened(const winrt::IInspectable&, const winrt::IInspec
 
     if (auto const teachingTipPeer = winrt::FrameworkElementAutomationPeer::FromElement(*this).try_as<winrt::TeachingTipAutomationPeer>())
     {
-        auto const appName = []()
+        auto const notificationString = [this]()
         {
-            if (auto&& package = winrt::ApplicationModel::Package::Current())
+            auto const appName = []()
             {
-                return package.DisplayName();
-            }
-            return winrt::hstring{};
-        }();
+                try
+                {
+                    if (auto&& package = winrt::ApplicationModel::Package::Current())
+                    {
+                        return package.DisplayName();
+                    }
+                }
+                catch (...) {}
```

Comment:
Totally down with this fix to unblock things, but we should follow up about having an API that doesn't require a caught exception here.
1) Add a comment here
2) Open a bug on whoever owns the app model API to add a no exception version of this query (and link the bug in the comment)